src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
```

Comment:
The function lacks error handling for invalid input. Consider adding a check to ensure that 'sourceString' is a string and not null or undefined.